app/models/client.rb

```diff
@@ -1,3 +1,11 @@
 # model for clients, this model can makes orders
 class Client < ApplicationRecord
+  # method creates new client if client with this phone doesn't exist
+  def self.current_client(parametrs)
+    client = find_by(phone: parametrs[:phone])
```

Comment:
@domenic99 what about `find_or_initialize_by`?
And what if a user want to change their email, for example?